Dockerfile

```diff
@@ -2,5 +2,4 @@ FROM python:3.6-alpine
 COPY . /app
 WORKDIR /app
 RUN apk update && apk add postgresql-dev postgresql-client gcc python3-dev musl-dev
-RUN pip install -r requirements.txt
```

Comment:
Why remove this line?

We can add both here and on `run.sh` in case of a update.

Reply:
I removed because it is not necessary to have it in two places.

Reply:
But it will make the container initialization take longer